deploy/kubernetes/overlays/prow-gke-release-staging-rc/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ bases:
 images:
 - name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
   newName: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver
-  newTag: "v0.6.1"
+  newTag: "v0.6.2"
```

Comment:
Should this be -gke.0?

Reply:
it can be either - the staging gcr has both.

Reply:
updated